ci/cloudbuild/build.sh

```diff
@@ -116,6 +116,21 @@ if [[ "${LOCAL_BUILD}" = "true" ]]; then
     print_usage
     exit 1
   fi
+  function mem_total() {
+    awk '$1 == "MemTotal:" {printf "%0.2f GiB", $2/1024/1024}' /proc/meminfo
+  }
+  io::log_h1 "Machine Info"
+  TZ=/etc/localtime printf "%10s %s (%s)\n" "local:" "$(date)" "$(date +%z)"
+  printf "%10s %s\n" "utc:" "$(date -u)"
```

Comment:
It might be nice if these were guaranteed to show the same time, with something like ...
```
now=$(date +%s)
TZ=/etc/localtime printf "%10s %s (%s)\n" "local:" "$(date -d @${now})" "$(date +%z -d @${now})"
printf "%10s %s\n" "utc:" "$(date -u -d @${now})"
```
And then you could also avoid the need for the extra `+%z` call by using RFC3339 output like ...
```
TZ=/etc/localtime printf "%10s %s\n" "local:" "$(date -d @${now} --rfc-3339=seconds)"
printf "%10s %s\n" "utc:" "$(date -u -d @${now} --rfc-3339=seconds)"
```
But, who cares about "localtime" (and whether `TZ=/etc/localtime` is the right way to force it)?  Perhaps we should be running all these scripts under `export TZ=:UTC`?

Reply:
> It might be nice if these were guaranteed to show the same time, with something like ...
> 
> ```
> now=$(date +%s)
> TZ=/etc/localtime printf "%10s %s (%s)\n" "local:" "$(date -d @${now})" "$(date +%z -d @${now})"
> printf "%10s %s\n" "utc:" "$(date -u -d @${now})"
> ```
> 
> And then you could also avoid the need for the extra `+%z` call by using RFC3339 output like ...
> 
> ```
> TZ=/etc/localtime printf "%10s %s\n" "local:" "$(date -d @${now} --rfc-3339=seconds)"
> printf "%10s %s\n" "utc:" "$(date -u -d @${now} --rfc-3339=seconds)"
> ```
> 
> But, who cares about "localtime" (and whether `TZ=/etc/localtime` is the right way to force it)? Perhaps we should be running all these scripts under `export TZ=UTC`?

*Our* logs are always in UTC, but others' log output is sometimes in localtime. We see this often with Kokoro where timestamps are mixed between UTC, localtime, and America/Los_Angeles. Even the GCB UI will sometimes show timestamps in localtime. The idea was to highlight to the reader of the logs whether or not localtime was different, and what the difference was.

But, I do like your suggestion of forcing UTC for everything, so I changed out GCBs and docker builds to set `TZ=:UTC` so that all logs will be in UTC (hopefully). The Cloud Build web UI will still show localtime sometimes.

I removed the print for localtime.

I added a line that logs the time that Google thinks it is, with the idea that this could help us diagnose time sync issues that may affect our integration tests (which we've seen happen on Kokoro).

See https://github.com/googleapis/google-cloud-cpp/pull/6124